### DISCUSSION TITLE
Storybook Angular 7 and Angular 16

### DISCUSSION THREAD

**Discussion by e-oz** [storybookjs/storybook#21882] [General]
```
@storybook/angular@7.0.0" has incorrect peer dependency "@angular/core@>=14.1.0 < 16.0.0".
```
Looks like you are going to release v7 pretty close to Angular v16 release, and versions will be incompatible. Please reconsider it.

**Comment by byjokese** [comment]
I guess v16 could take a couple of weeks/a month more easily, but yeah, it will be a huge request to be compatible, once it arrives. And I guess v16 will bring also Vite [compatibility](https://twitter.com/mgechev/status/1641512018263040001), so maybe we could have full Vite support on Storybook.
